pkg/cover/canonicalizer.go

```diff
@@ -103,18 +113,18 @@ func (can *Canonicalizer) NewInstance(modules []host.KernelModule) *Canonicalize
 	}
 }
 
-func (ci *CanonicalizerInstance) Canonicalize(cov []uint32, sign signal.Serial) {
+func (ci *CanonicalizerInstance) Canonicalize(cov *[]uint32, sign *signal.Serial) {
```

Comment:
Nit: I think it'd be cleaner to do `Canonicalize(cov []uint32, sign signal.Serial) ([]uint32, signal.Serial)`. Same for `Decanonicalize`.

Reply:
Sounds good! Done